ArticleTemplates/assets/js/modules/ads.js

```diff
@@ -58,12 +60,12 @@ function (
         for (i = 0; i < blocks.length; i++) {
             block = blocks[i];
 
-            if (i === 1 || i === 6) {
+            if (i === 2 || i === 7) {
                 numberOfMpus++;
                 mpu = createMpu(numberOfMpus);
 
                 if (block.nextSibling) {
-                    block.parentNode.insertBefore(mpu, block.nextSibling);
+                    block.parentNode.insertBefore(mpu, block);
```

Comment:
This will result in advert's going after the 3rd and 8th blocks, they should still go behind the 2nd and 7th. 

The only time they shouldn't is if an epic is between the 2nd/3rd and 7th/8th...then they should go between the next block immediately proceeded by a block.